org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_batch.scss

```diff
@@ -146,14 +152,137 @@
       border: none !important;
     }
 
+    .form-layout-compressed {
+      margin-bottom: 20px;
+
+      .label {   
+        color:  $crm-copy;
+      }
+
+      td.right {
+        color:  $crm-copy;            
+        vertical-align: middle;
+      }
+    }
+
+    .crm-grid-table {
+      border: none !important;
+      border-radius: $border-radius-base;
+      box-shadow: $crm-form-layout-shadow;
+
+      .crm-grid-header {
+        border-bottom: 1px solid  $crm-dark-grayblue;
+        border-top-left-radius: $border-radius-base;
+        border-top-right-radius: $border-radius-base;
+
+        .crm-grid-cell {
+          background: $gray-lighter;
+          border: none;
+          padding: 10px;
+          vertical-align: middle;
+
+          &:first-child {
```

Comment:
You shouldn't exceed 3 levels of hierarchy try to minimize this. maybe by using 

.crm-grid-header .crm-grid-cell {

or 
.crm-grid-cell:first-child